src/main/java/digital/slovensko/autogram/core/eforms/dto/EFormAttributes.java

```diff
@@ -34,8 +34,11 @@ private static EFormAttributes build(String identifier, String transformation, S
             }
         }
 
-        if (transformation != null)
+        if (transformation != null) {
             xsltParams = EFormUtils.fillXsltParams(transformation, identifier, xsltParams);
+            if (!transformation.isEmpty() && transformation.charAt(0) == '\uFEFF')
```

Comment:
Tieto vynimky mam pocit, ze nemame uplne pokryte v testoch a zacina sa to tu kopit. Vieme pridat aj nejaky test co overi, ze sa to vsetko nastavilo spravne?

Reply:
toto iba odstraňuje BOM zo začiatku stringu. Mali sme to na inom mieste, ale nie v API transformácii.